### PR TITLE
Isaicp 3137: Add "Solutions" and "Content" pages and menu items

### PR DESCRIPTION
**Pull request by idimopoulos**

### CODE REVIEW

File: tests/features/content_overview.feature [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    Then I should see the link "Content"
+    When I click "Content"
+    Then I should see the heading "Content"
+    # Check that all logged in users can see and access the link as well.
+    Given I am logged in as a user with the "authenticated user" role
+    Then I should see the link "Content"
+    When I click "Content"
+    Then I should see the heading "Content"
+
+  # @todo The small header, which contains content link, should be removed for anonymous users on the homepage - https://webgate.ec.europa.eu/CITnet/jira/browse/ISAICP-2639.
```
Comment (brummbar) [inline tests/features/content_overview.feature]: Line is too long, split it.

File: tests/features/solution/solution_overview.feature [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+      | Non electronic health | Supports health-related fields | validated |
+      | Closed data           | Facilitate access to data sets | validated |
+      | Isolating Europe      | Reusable tools and services    | validated |
+    Given the following owner:
+      | name              | type                    |
+      | NonProfit example | Non-Profit Organisation |
+    And the following contact:
+      | email | foo@bar.com         |
+      | name  | Information example |
+    # Check that visiting as an anonymous does not create cache for all users.
```
Comment (brummbar) [inline tests/features/solution/solution_overview.feature]: In the content test I added the moderator as the first user to visit the page. Do the same here.

File: tests/features/solution/solution_overview.feature [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+    And I press "Add owner"
+    And I press "Publish"
+    Then I visit the "Colonies in Earth" solution
+    Then I should see the text "Colonies in Earth"
+
+    And I am on the homepage
+    And I click "Solutions"
+    Then I should see the text "Colonies in Earth"
+
+    # @todo: Normally the solution should go through a moderation process.
```
Comment (brummbar) [inline tests/features/solution/solution_overview.feature]: What is this `@todo` for?

Comment (idimopoulos) [inline tests/features/solution/solution_overview.feature]: I initially intended to have it as a reminder that this should have been created by a non moderator user and be published by a moderator. But this is not part of this test so we can accept that it is directly published by a moderator.
Removing the comments.

File: tests/features/solution/solution_overview.feature (hunk 2 above)
Comment (idimopoulos) [inline tests/features/solution/solution_overview.feature]: Done.

File: tests/features/content_overview.feature (hunk 1 above)
Comment (idimopoulos) [inline tests/features/content_overview.feature]: Done.

**Pull request merged**